static/js/volunteer-schedule.js

```diff
@@ -74,36 +80,6 @@ function init_volunteer_schedule(data, all_roles, active_day) {
         };
     }
 
-    function make_modal_details(shift) {
-        var dl = $(document.createElement('dl')),
-            res = [
-                dl,
-                make_ele('p', '<a href="'+shift.sign_up_url +'">More details</a>')],
-            needed;
-
-        if (shift.min_needed === shift.max_needed) {
-            needed = shift.min_needed;
-        } else {
-            needed = shift.min_needed + ' - ' + shift.max_needed;
-        }
-
-        if (shift.is_user_shift) {
-            res.unshift(make_ele('p', 'You are currently signed up to this shift'));
```

Comment:
Is this functionality no longer needed?

Reply:
The same functionality is provided by the sign up buttons, which make it clearer in the list view that you're signed up for a given shift. Volunteer counts are also displayed there.